src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -1440,6 +1490,23 @@ export default class DefaultDeviceController
         kind,
         this.getMediaTrackSettings(newDevice.stream)?.deviceId || ''
       );
+
+      if (kind === 'audio') {
+        // We only monitor the first track, and use its device ID for observer notifications.
+        const track = newDevice.stream.getAudioTracks()[0];
+        if (track) {
+          const id = track.getSettings().deviceId || newDevice.stream;
+
+          newDevice.trackMuteCallback = (): void => {
+            this.mediaStreamMuteObserver(id, true);
+          };
+          newDevice.trackUnmuteCallback = (): void => {
+            this.mediaStreamMuteObserver(id, false);
+          };
+          track.addEventListener('mute', newDevice.trackMuteCallback, { once: false });
```

Comment:
Isn't the default is false for `once`? Do you just want to be explicit here?

Reply:
Yes, just being explicit.